applications/MappingApplication/tests/cpp_tests/test_closest_points.cpp

```diff
@@ -63,17 +63,17 @@ KRATOS_TEST_CASE_IN_SUITE(PointWithIdEqualComparison, KratosMappingApplicationSe
     PointWithId point_2(id, coords, dist);
     PointWithId point_3(id, coords2, dist);
 
-    // only the position aka the coordinates are used for the equal comparison!
     PointWithId point_4(id+1, coords, dist);
     PointWithId point_5(id, coords, dist+1.0);
     PointWithId point_6(id+1, coords, dist+1.0);
 
+    // Coordinates and Distance (id excluded) are used for the equal comparison!
     KRATOS_EXPECT_EQ(point_1, point_2);
     KRATOS_EXPECT_NE(point_1, point_3);
```

Comment:
do you know what it was using before since I did not have the other operators defined?

Reply:
probably sth from the baseclass(es) no?

But should it not be able to derive these checks from operator<? I vaguely remember that was somewhat important for the spaceship operator no?

Reply:
`<` does not allow evaluation of `==` or `!=` until c++20 that I remember, so it must have been inheriting from`Point` or `IndexedObject`. Since the comment `only the position aka the coordinates are used for the equal comparison!` and the test for 1 vs 3 was set to EQ and was passing, I assume it was inheriting from `Point`

Reply:
Well this is why I was asking, so, revert KRATOS_EXPECT_NE ti KRATOS_EXPECT_EQ, but then ignore the distance in the new `==` operator?

Reply:
exactly!